examples/vllm-gcs/config.yaml

```diff
@@ -0,0 +1,22 @@
+build:
+  arguments:
+    endpoint: Completions
+    model: /app/hf_cache/llama-2-7b
```

Comment:
can we just make the model `gs://varuns-llama2-whatever` and then do the swap for the args under the hood so the user doesn't have to worry about it and to add the `hf_cache` option below as well?

Reply:
What if the model was a private HF cache? What would the expected behavior be then?